src/Features/Lsif/GeneratorTest/CompilerInvocationTests.vb

```diff
@@ -0,0 +1,173 @@
+﻿' Licensed to the .NET Foundation under one or more agreements.
+' The .NET Foundation licenses this file to you under the MIT license.
+' See the LICENSE file in the project root for more information.
+
+Imports Microsoft.CodeAnalysis.Test.Utilities
+
+Namespace Microsoft.CodeAnalysis.LanguageServerIndexFormat.Generator.UnitTests
```

Comment:
💡 It would be good to have these as C# code from the start

Reply:
Yeah, the XML Workspace syntax is too useful to ignore. The tooling for embedding it in a string is almost unusable. 😦